python/vowpalwabbit/pyvw.py

```diff
@@ -1249,14 +1249,55 @@ def from_example(ex: "Example"):
                 ex.get_cbandits_probability(i),
             )
             costs.append(cb)
-        return CBLabel(costs, prediction)
+        return CBLabel(costs, weight)
 
     def __str__(self):
         return " ".join(
             ["{}:{}:{}".format(c.action, c.cost, c.probability) for c in self.costs]
         )
 
 
+class CBEvalLabel(AbstractLabel):
+    """Class for contextual bandits eval VW label"""
+
+    def __init__(
+        self,
+        action: int = 0,
+        cb_label: CBLabel = CBLabel(),
```

Comment:
I kind of don't think all of the labels should have these defaults. When would a user actually need to create a label is some default state?

Reply:
You think it would be better to make it Optional?

Reply:
Why optional? I think the values should be required without defaults

Reply:
I set action to 0 to be consistent with C++ but I agree that isnt a good default

Reply:
Since the primary way of creation in python is from an example object we don't need the defaults for anything in particular. I was just thinking as a way to make it harder to create a potentially nonsensical default object. This is really not a big deal though

Reply:
Makes sense I got rid of the default